scripts/collect-datasets.js

```diff
@@ -114,10 +123,15 @@ function filenameLooksLikeDataset(filename) {
   if (!filename.endsWith(".json")) return false;
   if (filename.endsWith("_meta.json") || filename.endsWith("_tree.json")) return false;
   if (filename.endsWith("_frequencies.json")) return false;
+  if (filename.endsWith("_titers.json")) return false;
+  if (filename.endsWith("_frequencies.json")) return false;
   if (filename.endsWith("_tip-frequencies.json")) return false;
   if (filename.endsWith("_aa-mutation-frequencies.json")) return false;
   if (filename.endsWith("_sequences.json")) return false;
   if (filename.endsWith("_entropy.json")) return false;
   if (filename.endsWith("_root-sequence.json")) return false;
+  if (filename.includes("/")) return false;
```

Comment:
Could you add a comment to this line indicating that S3 objects (files) in folders cannot be accessed by Auspice, and therefore are ignored by this script.